<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="widget_section_name">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_percentage_correct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>xx % correct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="widget_question_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Question</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="widget_answer_table">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Correct?</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Answer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_previous">
       <property name="text">
        <string>&lt;&lt; Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_next">
       <property name="text">
        <string>Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save &amp;&amp; Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_previous</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>previous_section()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next_section()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_answer_table</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>Dialog</receiver>
   <slot>cell_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>previous_section()</slot>
  <slot>next_section()</slot>
  <slot>cell_changed()</slot>
  <slot>close_window()</slot>
 </slots>
</ui>
